agent/command/git.go

```diff
@@ -213,6 +213,31 @@ func (opts cloneOpts) buildSSHCloneCommand() ([]string, error) {
 	}, nil
 }
 
+func moduleRevExpansionName(name string) string { return fmt.Sprintf("%s_rev", name) }
+
+// Load performs a GET on /manifest/load
+func (c *gitFetchProject) manifestLoad(ctx context.Context,
```

Comment:
It looks to me like this can be a helper function that isn't a method on gitFetchProject, since it doesn't use anything within c.

Reply:
I did that because otherwise when it's called above, it automatically references [this struct](https://github.com/evergreen-ci/evergreen/blob/main/agent/command/manifest.go#L15) and not the function. Making it part of c felt cleaner than messing with the name.